cms-derived-data/config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# all recids are placeholders, should be changed later to correct ones
+common_values:
+  abstract:
+    description: <p>Dataset in reduced NanoAOD format for education and outreach derived from the primary dataset in AOD format linked below.</p>
+  accelerator: CERN-LHC
+  authors: 
+    - CMS Open Data Group
+  collections:
+    - CMS-Derived-Datasets
+  control_number: 12365
+  dataset_semantics:
+  date_published: 2023
+  distribution:
+    formats:
+      - root
+      - nanoaod
+  experiment: CMS
+  license:
+    attribution: CC0
+  publisher: CERN Open Data Portal
+  type:
+    primary: Dataset
+    secondary:
+      - Derived
+  validation:
+    description: <p>These data were derived from the primary datasets linked below using only the validated runs. No further validation was done for the output.</p>
```

Comment:
This will be different for POET, Run1 and PF. This is OK for POET. For Run1, "No further validation was done for the output." can be removed.
For pfnano, everything was processed, not only validated runs. we'll check with @jmhogan how to formulate this.